tests/app/validation/test_textarea_type_check.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+# from app.schema.answer import Response
+# from app.schema.question import Question
+# from app.schema.section import Section
+# from app.schema.block import Block
+# from app.schema.questionnaire import Questionnaire
+# from app.schema.group import Group
+from app.validation.textarea_type_check import TextAreaTypeCheck
+from app.validation.abstract_validator import AbstractValidator
+
+
+class TextAreaTest(unittest.TestCase):
+
+    def test_textarea_validator(self):
+        textarea = TextAreaTypeCheck()
+
+        # validate integer
+        result = textarea.validate(1)
+        self.assertFalse(result.is_valid)
+        self.assertEquals(len(result.errors), 1)
+        self.assertEquals(AbstractValidator.NOT_STRING, result.errors[0])
+
+        # validate None
+        result = textarea.validate(None)
+        self.assertFalse(result.is_valid)
+        self.assertEquals(len(result.errors), 1)
+        self.assertEquals(AbstractValidator.NOT_STRING, result.errors[0])
+
+        # validate <space>
+        result = textarea.validate(' ')
+        self.assertFalse(result.is_valid)
```

Comment:
There are no errors for empty spaces; if it's mandatory it will be caught by mandatory error, otherwise answer won't be stored.